src/platform/components/Header/index.tsx

```diff
@@ -8,20 +8,41 @@ import Logo from "assets/img/logo.png";
 
 type HeaderProps = {
   backButtonEnabled?: boolean;
+  navsEnabled?: boolean;
 };
 
-const Header: FC<HeaderProps> = ({ backButtonEnabled }) => {
+const Header: FC<HeaderProps> = ({
+  backButtonEnabled = false,
```

Comment:
No need to set a default value for a bool type, because its undefined by default "falsy"

Reply:
Done